cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/stage/SvCalling.java

```diff
@@ -179,7 +178,13 @@ private BashCommand buildRefDepthCommand(final String gridssVcf, final OutputFil
         // arguments.add("-log_level DEBUG");
         arguments.add(format("-threads %s", Bash.allCpus()));
 
-        return new JavaClassCommand(SV_PREP, SV_PREP_DEPTH_ANNOTATION, arguments);
+        return new JavaClassCommand(SV_PREP.getToolName(),
```

Comment:
I think I can add this and it would be nice for `SvCalling` in particular but for the other usages it wouldn't work because: 

1. They directly inherit from `JavaClassCommand`.
2. In `CramAndValiateCommands` it uses the `BAMCOMP` constant which is not a `HmfTool` but an `ExternalTool` which has a slightly different interface (although I could make a seperate factory method for this interface as well.)